doc/DYNAMIC_COMPILER_FORMATS.md

```diff
@@ -193,3 +193,28 @@ Session completed
 See `Sha256.java` from `https://github.com/AuthMe/AuthMeReloaded` for addtional details.
 
 NOTE: The inbuilt `dynamic_65` format is a bit faster at cracking such hashes.
+
+## Cracking BitShares (versions <= 0.9.3c) hashes
+
+Extract the `checksum` data from the `~/.BitShares/wallets/<wallet-name>/*.ldb` file using `bitshares2john.py` program.
+
+```
+$ cat sample-hashes
+b335d77abf42f6da1b6d0864129fa176c7c499b70c389b273e719473e5f029f54b9b8914e96a6585db2f16c987e6b988e78a0e497bbe8bc88d62df0eb81b7010
+f4bf2f7ba63b375dd21325f06b3b857294d88e760d854125459d4f1b82bf576d2de53ab0f52fd96990bd6ee7cee956c4bf133d79b7c1575cc4624ca0d3da7be4
+```
+
+```
+$ ../run/john -form=dynamic='sha512(sha512_raw($p))' sample-hashes -w=wordlist
+Using default input encoding: UTF-8
+Loaded 2 password hashes with no different salts (dynamic=sha512(sha512_raw($p)) [256/256 AVX2 4x])
+Press 'q' or Ctrl-C to abort, almost any other key for status
+password123      (?)
+openwall         (?)
+```
+
+NOTE: The inbuilt `dynamic_84` format is a bit faster at cracking such hashes. It can be used via the following command,
+
+```
+$ ../run/john --format=dynamic_84 sample-hashes -w=wordlist
+```
```

Comment:
Why do we even suggest the `-form=dynamic='sha512(sha512_raw($p))'` if dynamic_84 is faster?

Reply:
You have a point. I will remove this part.